wafer/talks/templates/wafer.talks/talk.html

```diff
@@ -108,6 +108,11 @@ <h1>
         {% else %}
           <span class="badge badge-danger">{% trans 'Not accepted' %}</span>
         {% endif %}
+        {% if object.is_late_submission %}
+        {% with submission=object.submission_time %}
+        <span class="badge badge-warning ">{% blocktrans %}Late submission ({{ submission }}){% endblocktrans %}</span>
```

Comment:
Should we display the submission time somewhere even if the talk was not submitted late?

Reply:
good idea